mmseg/datasets/pipelines/transforms.py

```diff
@@ -240,6 +244,19 @@ def _random_scale(self, results):
     def _resize_img(self, results):
         """Resize images with ``results['scale']``."""
         if self.keep_ratio:
+            if self.setr_multi_scale:
```

Comment:
I think this if condition can be
```suggestion
            if self.crop_size is not None:
```
And we only add a crop_size in `__init__`  arg list
Add some comments like `# keep the shape is not less than crop_size`